Set hystrix config failed

<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!

-->


**What happened**:
After setting the hystrix configuration, a panic error occurred during startup
```
panic: interface conversion: interface {} is map[string]interface {}, not map[interface {}]interface {}

goroutine 42 [running]:
dubbo.apache.org/dubbo-go/v3/filter/hystrix.initConfigConsumer()
        /Users/xxxx/go/pkg/mod/dubbo.apache.org/dubbo-go/v3@v3.0.1/filter/hystrix/filter.go:254 +0x29b
```

**What you expected to happen**:
The program starts normally and hystrix takes effect

**How to reproduce it (as minimally and precisely as possible)**:
Use this yaml:
```
dubbo:
  consumer:
      filter: hystrix_consumer
      filter-conf:
        hystrix:
          configs:
            Default:
              timeout: 100
              sleep_window: 5
              max_concurrent_requests: 5000
              error_percent_threshold: 1
              request_volume_threshold: 1
          default: "Default"
  references:
      IndexClientImpl:
        protocol: grpc
        interface: com.xxx.index.server
        group: xxx_group
        version: v1.0
        check: false
        retries: 0
        async: true
```

**Anything else we need to know?**: 
dubbo-go v3.0.1

Can u provide a demo configuration of hystrix please?

--- a/filter/hystrix/filter.go
+++ b/filter/hystrix/filter.go
@@ -21,6 +21,7 @@ package hystrix
 import (
 	"context"
 	"fmt"
+	"reflect"
 	"regexp"
 	"sync"
 )
@@ -252,7 +253,14 @@ func initConfigConsumer() error {
 	if config.GetConsumerConfig().FilterConf == nil {
 		return perrors.Errorf("no config for hystrix_consumer")
 	}
-	filterConfig := config.GetConsumerConfig().FilterConf.(map[interface{}]interface{})[HYSTRIX]
+	filterConfig_ := config.GetConsumerConfig().FilterConf
+	var filterConfig interface{}
+	switch reflect.ValueOf(filterConfig_).Interface().(type) {
+	case map[interface{}]interface{}:
+		filterConfig = config.GetConsumerConfig().FilterConf.(map[interface{}]interface{})[HYSTRIX]
+	case map[string]interface{}:
+		filterConfig = config.GetConsumerConfig().FilterConf.(map[string]interface{})[HYSTRIX]
+	}
 	if filterConfig == nil {
 		return perrors.Errorf("no config for hystrix_consumer")
 	}

--- a/filter/hystrix/filter.go
+++ b/filter/hystrix/filter.go
@@ -253,9 +253,9 @@ func initConfigConsumer() error {
 	if config.GetConsumerConfig().FilterConf == nil {
 		return perrors.Errorf("no config for hystrix_consumer")
 	}
-	filterConfig_ := config.GetConsumerConfig().FilterConf
+	filterConf := config.GetConsumerConfig().FilterConf
 	var filterConfig interface{}
-	switch reflect.ValueOf(filterConfig_).Interface().(type) {
+	switch reflect.ValueOf(filterConf).Interface().(type) {
 	case map[interface{}]interface{}:
 		filterConfig = config.GetConsumerConfig().FilterConf.(map[interface{}]interface{})[HYSTRIX]
 	case map[string]interface{}:

